tests/data/preview/prefer_rhs_split.py

```diff
@@ -50,7 +50,7 @@
     forth_item,
     fifth_item,
     last_item_very_loooooong,
-) = everyting = some_loooooog_function_name(
+) = everything = some_loooooog_function_name(
```

Comment:
```suggestion
) = everything = some_looooong_function_name(
```
While we're changing this